spinta/core/config.py

```diff
@@ -643,7 +644,11 @@ def configure_rc(
         }
         config['manifest'] = 'default'
 
+        config['check.manifests'] = manifests
+
+        if check_filename is not None:
+            config['check.filename'] = check_filename
```

Comment:
This adds configuration file only if manifest file is specified via command line arguments, but this should be enabled in all cases, no matter how manifest is configured.

Also, we probably need to set root directory of manifest files, this could be done by adding `-C, --directory` (see [git](https://git-scm.com/docs/git#Documentation/git.txt--Cltpathgt), [tar](https://www.mankier.com/1/tar#-C)) flag to all commands, similarly how it is done with `-o` in `spinta/cli/main.py`.

This way command should look like this:

    spinta -C path/to/root check --check-filename

Or same thing can be set via configuration parameters:

    spinta -o directory=path/to/root check --check-filename